tests/ident/test_engine_parser_omssa.py

```diff
@@ -146,7 +150,8 @@ def test_engine_parsers_omssa_is_iterable():
             "Raw data location": "/Users/cellzome/Dev/Gits/Ursgal/ursgal_master/example_data/test_Creinhardtii_QE_pH11.mgf",
         },
     )
-    assert isinstance(parser, Iterable)
+    df = parser.unify()
+    assert len(df) == 179
```

Comment:
same as above - please also assert identity of the resulting df - not just first line :)